src/main/flight/gps_rescue.c

```diff
@@ -142,11 +142,11 @@ void gpsRescueInit(void)
     rescueState.sensor.gpsRescueTaskIntervalSeconds = HZ_TO_INTERVAL(TASK_GPS_RESCUE_RATE_HZ);
 
     float cutoffHz, gain;
-    cutoffHz = positionConfig()->altitude_d_lpf / 100.0f;
+    cutoffHz = positionConfig()->altitude_d_lpf * 0.01f;
```

Comment:
This is initialization code, so speed doesn't matter here. It can be `* 0.01` or `/ 100.0f`, doesn't matter.

Reply:
@ctzsnooze : Did you pass -ffast-math to compiler?

Reply:
I used the settings from your godbolt link https://godbolt.org/z/hM6YYWjea
It has `-O3 -mthumb -mcpu=cortex-m4 ` options as per your suggestion.  See:
https://godbolt.org/z/bY4Yo6d18
Do I need different settings?

Reply:
Yes, pretty sure. See https://github.com/betaflight/betaflight/blob/master/Makefile#L152-L155

Reply:
I'll reset them all to simple `/ 100.0f` format where it's easier to read.